src/components/budget/budget.service.ts

```diff
@@ -13,6 +12,7 @@ import {
   ConfigService,
   createBaseNode,
   DatabaseService,
+  permission as dbPermission,
```

Comment:
There's no need to rename the import. If we don't like the name we should just change it instead of fighting the default in every file.